_data/simple-icons.json

```diff
@@ -1898,7 +1898,7 @@
         {
             "title": "Conda-Forge",
             "hex": "000000",
-            "source": "https://github.com/conda-forge/conda-forge.github.io/"
+            "source": "https://github.com/conda-forge/conda-forge.github.io/blob/34adb68298ca266af13c3d615f7af8b6c232f6fb/img/anvil.svg"
```

Comment:
It seems that you slipped this change from another PR 🙃

Reply:
yes. reverted this already.